src/io/util.cpp

```diff
@@ -0,0 +1,37 @@
+#include "aikido/io/util.hpp"
+
+namespace aikido {
+namespace io {
+
+dart::dynamics::SkeletonPtr loadSkeletonFromURDF(
```

Comment:
Nit: We typically put "//==================================" at the beginning of a new method. Please check other cpp files.